ui/src/app/views/workspace/component.ts

```diff
@@ -129,12 +133,25 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
       this.researchPurposeArray.slice(
         this.leftResearchPurposes.length,
         this.researchPurposeArray.length);
-    this.showTip = true;
+    this.showTip = false;
+    this.newPageVisit.page = WorkspaceComponent.PAGE_ID;
   }
 
   ngOnInit(): void {
     this.wsNamespace = this.route.snapshot.params['ns'];
     this.wsId = this.route.snapshot.params['wsid'];
+    this.profileService.getMe().subscribe(
```

Comment:
tl;dr: I would add a TODO here to switch this to use the profileStorageService.

This should probably use the profileStorageService, which caches the profile lookup. *But* unfortunately this might cause bugs in your help tip display within a single client session because the profile doesn't get updated dynamically on methods that mutate the profile. I filed a ticket to make that happen https://precisionmedicineinitiative.atlassian.net/projects/RW/issues/RW-1057

Reply:
+1